jenkins.groovy

```diff
@@ -62,6 +62,33 @@ pipeline
               echo "MINOR: ${MINOR}"
               echo "PATCH: ${PATCH}"
 
+              //Is the revision suffix just a number?
+              if (TAG_NAME =~ /^([4-9]|\d\{2,\}+).([0-9]+).([0-9]+)-([0-9]+)$/)
```

Comment:
Escaping the curly braches doesn't work. (at least in Java) Also, the `+` quantifier is not needed because the `{2,}` is already a quantifier
```suggestion
              if (TAG_NAME =~ /^([4-9]|\d{2,}).([0-9]+).([0-9]+)-([0-9]+)$/)
```

Reply:
I copied the regex from our Docker Hub build, and the escaped curly brace was the only way to make it work in Docker Hub. 

It worked this way when I did the beta release. I will test it without the escapes and `+` and see how it goes.